packages/patternfly-3/patternfly-react/src/components/Chart/DonutChart.js

```diff
@@ -15,13 +15,14 @@ const setDonutTitle = obj => {
 
   const { props } = obj;
   const { data, title = {} } = props;
+  const { type, percision = 0 } = title;
   const { columns } = data;
   const sum = columns.reduce((acc, x) => acc + x[1], 0);
   const iMax = colIndexOfMaxValue(columns);
 
-  switch (title.type) {
+  switch (type) {
     case 'percent':
-      primary = `${Math.round((100 * columns[iMax][1]) / sum).toString()}%`;
+      primary = `${((100 * columns[iMax][1]) / sum).toFixed(percision).toString()}%`;
```

Comment:
actually this may still be a problem - (100 * 99.99999999 / 100).toFixed(2) # => 100.00
I'm not sure if the other suggestion solves it, maybe it does